spec/models/invoice_spec.rb

```diff
@@ -47,4 +51,14 @@
       end
     end
   end
+
+  describe 'class methods' do
+    describe '.incomplete_invoices' do
+      it 'returns invoices only with items that are packaged or pending' do
+        invoices = Invoice.all
+        expect(invoices.incomplete_invoices).to eq([invoice_1, invoice_2])
```

Comment:
Could we just call `expect(Invoice.incomplete_invoices).to eq([invoice_1, invoice_2])` to clean up a line?

Reply:
Yeah, we probably could. For some reason when I was doing that in the tests though it wasn't working. :-/ Not entirely sure why. It was saying that I asked for an array but got an active record relation when I just called it on the class, but when I called it on an instance representing the entire class I got what I was looking for. And I could see in pry that it was returning what I wanted. In other tests we didn't have that same problem, and the syntax waqs the same, so I couldn't figure it out. But I can definitely try it again. :)

Reply:
Lol. So, I am furious. I went back into my code and tried it your way and this time it worked. So I will make that change.

Reply:
Updated in the appended commit.